### PR TITLE
Restrict text_on_statement

### PR DESCRIPTION
**Pull request by ct-clearhaus**

### CODE REVIEW

**Comment by ct-clearhaus** [comment]
> If we reject based on a rule involving text on statement will we also reject with status code "Invalid text on statement"?

Nope. Then it's a rule violation. The invalid text on statement is one that doesn't follow the API spec. This is common for all merchants whereas the rules differ between merchants.

**Comment by soltveit** [comment]
Right, makes sense.

File: source/index.md [hunk 1]
```diff
@@ -540,6 +542,7 @@ POST https://gateway.clearhaus.com/authorizations/:id/refunds
   <dd>
     [\x20-\x7E]{0,22}
     <i><a target="_blank" href="http://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters">ASCII printable characters</a></i> </br>
+    <i>May not be all digits, all same character, or all sequential characters (e.g. "abc").</i></br>
     <i>Optional</i> <br />
     Text that will be placed on cardholder's bank statement.
```
Comment (mt-clearhaus) [inline source/index.md]: Maybe add "Overrides <code>text_on_statement</code> from authorization."?

Comment (ct-clearhaus) [inline source/index.md]: Please feel free to push 🙂

**Comment by ct-clearhaus** [comment]
Last push due to https://github.com/clearhaus/gateway-api-docs/pull/43#issuecomment-296962040

**Comment by ct-clearhaus** [comment]
How much time does PSPs want in order to handle an update like this to add a response status code?

I've checked the last 30 days of transactions finding only very few transactions that would be hit by this; I've informed you who've sent us those transactions.

Cc: @nielsek @ta @tjconcept @storkie

**Comment by ta** [comment]
@ct-clearhaus: around 0 :)

**Comment by tjconcept** [comment]
It's gracefully handled already. No immediate changes required here.

**Comment by mt-clearhaus** [comment]
Deployed in production today in the timespan 15:15-16:00 CEST.

**Pull request merged**